<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel::frontend::QuickKDESU</class>
 <widget class="QWidget" name="panel::frontend::QuickKDESU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>121</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="kdesu_panel_frame">
     <property name="styleSheet">
      <string notr="true">#kdesu_panel_frame {
  background: #eff1f5;
  border-radius: 8px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="kdesu_dolphin_btn">
        <property name="styleSheet">
         <string notr="true">#kdesu_dolphin_btn {
  color: #4c4f69;
  background: transparent;
  border: 0px solid #4c4f69;
  border-radius: 6px;
  font-size: 16px;
  text-align: left;
  padding: 6px;
}

#kdesu_dolphin_btn:hover {
  color: #dc8a78;
  background: #f5e0dc;
  border: 0px solid #dc8a78;
  border-radius: 6px;
  font-size: 16px;
  padding: 6px;
}

#kdesu_dolphin_btn:pressed {
  color: #eff1f5;
  background: #dc8a78;
  border: 0px solid #4c4f69;
  border-radius: 6px;
  font-size: 16px;
  padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Dolphin</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonStyle::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="kdesu_konsole_btn">
        <property name="styleSheet">
         <string notr="true">#kdesu_konsole_btn {
  color: #4c4f69;
  background: transparent;
  border: 0px solid #4c4f69;
  border-radius: 6px;
  font-size: 16px;
  text-align: left;
  padding: 6px;
}

#kdesu_konsole_btn:hover {
  color: #dc8a78;
  background: #f5e0dc;
  border: 0px solid #dc8a78;
  border-radius: 6px;
  font-size: 16px;
  padding: 6px;
}

#kdesu_konsole_btn:pressed {
  color: #eff1f5;
  background: #dc8a78;
  border: 0px solid #4c4f69;
  border-radius: 6px;
  font-size: 16px;
  padding: 6px;
}

#kdesu_konsole_btn:pressed {
  color: #eff1f5;
  background: #dc8a78;
  border: 0px solid #4c4f69;
  border-radius: 6px;
  font-size: 16px;
  padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Konsole</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::Computer"/>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonStyle::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
